### PR TITLE
Fixed fullname call in get_modifiedby

### PR DESCRIPTION
**Pull request by richardjonesnz**
Hi Davo

Thanks for the advice, hope this one works better.  Wasn't sure about changing version.php - I have only tested it in 3.10.1.  

Richard

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -688,7 +688,7 @@ public function get_modifiedby() {
             if ($this->modifiedbyid == $USER->id) {
                 $this->modifiedby = $USER;
             } else {
-                $this->modifiedby = $DB->get_record('user', array('id' => $this->modifiedbyid), 'id, firstname, lastname');
+                $this->modifiedby = $DB->get_record('user', array('id' => $this->modifiedbyid), get_all_user_name_fields(true));
```
Comment (davosmith) [inline lib.php]: You still need the 'id' field here - so: ```$DB->get_record('user', array('id' => $this->modifiedbyid), 'id, '.get_all_user_name_fields(true));```

**Comment by davosmith** [comment]
Bumping the version number is OK - I've been caught out before by publishing a plugin without a version number chance (because it didn't "technically" need it, as there was no DB upgrade needed); it causes confusion if it isn't changed.

**Comment by richardjonesnz** [comment]
OK, one more time.  What I meant about version number was I'm not sure about the etiquette of changing someone else's version number sequences.

**Comment by davosmith** [comment]
Thanks for your patience with my feedback - looks spot on, so I've merged it in.

I'm happy for you to update the version number - it means I don't have to remember to do it (but thanks for checking!)

**Comment by richardjonesnz** [comment]
Your feedback is much appreciated.

**Pull request merged**